integration/debug_test.go

```diff
@@ -71,3 +74,54 @@ func TestDebug(t *testing.T) {
 		})
 	}
 }
+
+func TestDebugEventsRPC(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+	if RunOnGCP() {
+		t.Skip("skipping test that is not gcp only")
+	}
+
+	rpcAddr := randomPort()
+
+	// Run skaffold build first to fail quickly on a build failure
+	skaffold.Build().InDir("testdata/jib").RunOrFail(t)
+
+	ns, client, deleteNs := SetupNamespace(t)
+	defer deleteNs()
+
+	stop := skaffold.Debug("--enable-rpc", "--rpc-port", rpcAddr).InDir("testdata/jib").InNs(ns.Name).RunBackground(t)
+	defer stop()
+
+	client.WaitForPodsReady()
+
+	_, entries, shutdown := apiEvents(t, rpcAddr)
+	defer shutdown()
+
+	metaEntries, buildEntries, deployEntries, debuggingEntries := 0, 0, 0, 0
+	timeout := time.After(1 * time.Minute)
+	for i := 0; i < 6; i++ {
```

Comment:
this is brittle, what if we change events? 
it also did fail with the new status events introduced

Reply:
I did change it.